TypeCobol.Test/Parser/Programs/Cobol85/SpecialNamesParagraph.rdz.cbl

```diff
@@ -0,0 +1,28 @@
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. TCOMFL06.
+       ENVIRONMENT DIVISION.
+       CONFIGURATION SECTION.
+       special-names.
+           decimal-point comma
+           SYMBOLIC CHARACTERS LOW-VAL IS 1
+                               HEX-15  IS 22
+                               HI-VAL  IS 256
+           CLASS NAMECHAR IS 'A' THRU 'I'
+                             'J' THRU 'R'
+                             'S' THRU 'Z'
+           .
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+       01 var1 PIC X(10) VALUE LOW-VAL.
+       01 var2 PIC X(10) VALUE HEX-15.
+       01 var3 PIC X(10) VALUE HI-VAL.
+       01 t1 PIC X(10).
+       01 t2 PIC X(10).
+       01 t3 PIC X(10).
+       01 c PIC X.
+       PROCEDURE DIVISION.
+           MOVE 'M' TO c
+           COPY CopyUsingSpecialNames.
+           GOBACK
+           .
+       END PROGRAM TCOMFL06.
```

Comment:
I suggest the following code sample to test that `SYMBOLIC CHARACTERS` is well supported (provided the execution output)
```cob
      
       IDENTIFICATION DIVISION.
       PROGRAM-ID. TCOMFL06.
       ENVIRONMENT DIVISION.
       CONFIGURATION SECTION.
       special-names.
           decimal-point comma
           SYMBOLIC CHARACTERS LOW-VAL IS 1
                               HEX-15  IS 22
                               HI-VAL  IS 256
           CLASS NAMECHAR IS 'A' THRU 'I'
                             'J' THRU 'R'
                             'S' THRU 'Z'
           .
       DATA DIVISION.
       WORKING-STORAGE SECTION.
       01 var1 PIC X(10).
       01 var2 PIC X(10).
       01 var3 PIC X(10).
       01 c    PIC X.
       PROCEDURE DIVISION.
      *   Test NAMECHAR
           MOVE 'M'      TO c
           IF c IS NAMECHAR
              DISPLAY "TEST NAMECHAR OK"
           ELSE
              DISPLAY "TEST NAMECHAR FAILED"
           END-IF
      *   Test LOW-VAL
           MOVE 1        TO var1
           IF var1 EQUAL LOW-VAL
              DISPLAY "TEST LOW-VAL OK"
           ELSE
              DISPLAY "TEST LOW-VAL FAILED"
           END-IF
      *   Test HEX-15
           MOVE 22       TO var2
           IF var2 EQUAL HEX-15
              DISPLAY "TEST HEX-15 OK"
           ELSE
              DISPLAY "TEST HEX-15 FAILED"
           END-IF
      *   Test HI-VAL
           MOVE 256      TO var3
           IF var3 EQUAL HI-VAL
              DISPLAY "TEST HI-VAL OK"
           ELSE
              DISPLAY "TEST HI-VAL FAILED"
           END-IF
           GOBACK
           .
```